src/journal-remote/journal-remote-main.c

```diff
@@ -152,6 +155,26 @@ static int dispatch_http_event(sd_event_source *event,
                                uint32_t revents,
                                void *userdata);
 
+static int build_accept_encoding(char **ret) {
+        assert(ret);
```

Comment:
as per coding style asserts go after variable declarations, with empty line before and after